packages/kysely/src/codes/get.ts

```diff
@@ -12,8 +12,7 @@ export function get(db: Kysely<Database>) {
     let query = db
       .selectFrom("codes")
       .where("codes.code_id", "=", code_id)
-      .where("codes.code_type", "=", code_type)
-      .where("codes.expires_at", ">", new Date().toISOString());
+      .where("codes.code_type", "=", code_type);
```

Comment:
Should I throw an error here instead?

Reply:
The downside with filtering in this case is that it's not possible to inform the use that the code is expired. I'm also planning to have multiple datbase adapters, so I try to avoid keeping logic in the adapter.